content/enterprise_influxdb/v1.9/administration/configure-password-hashing.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Configure password hashing
+description: >
+  Configure the cryptographic algorithm used for password hashing.
+menu:
+  enterprise_influxdb_1_9:
+    name: Configure password hashing
+    weight: 80
+    parent: Administration
+related:
+    - /enterprise_influxdb/v1.9/administration/configuration/
+---
+
+
+By default, InfluxDB Enterprise uses `bcrypt` for password hashing.
+[FIPS] compliance requires particular hashing alorithms.
```

Comment:
@pierwill did you mean to add a link for FIPS?

Reply:
the link is in on line 65